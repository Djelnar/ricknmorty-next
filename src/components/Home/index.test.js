/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import { Home } from './index'
import { Title } from './Title'


describe('Home Component', () => {
  it('should render correctly', () => {
    const output = renderer.create((
      <Home>
        <Title>Home Title</Title>
      </Home>
    )).toJSON()

    expect(output).toMatchSnapshot()
  })
})
