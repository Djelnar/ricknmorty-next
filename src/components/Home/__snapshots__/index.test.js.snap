// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render correctly 1`] = `
.c0 {
  height: calc(100vh - 52px);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: pink;
}

.c1 {
  font-size: 28px;
  color: #ffffff;
  text-align: center;
}

<main
  className="c0"
>
  <h1
    className="c1"
  >
    Home Title
  </h1>
</main>
`;
